keyvalue/blob/bytes.go

```diff
@@ -16,14 +16,14 @@ var (
 		SetBlob
 		GrowBlob
 		TruncateBlob
-	} = &Bytes{}
+	} = (*Bytes)(nil)
```

Comment:
This change isn't related, but is a (very mild) refactoring that lets the compiler remove this (I think we save 8 bytes of memory) - it's a little more idiomatic to do it this way.